### PR TITLE
Docs: Updating monorepo.md with extra mirror repo information

### PR DESCRIPTION
**Pull request by coder-karen**
#### Changes proposed in this Pull Request:

* Updating monorepo.md with a few additional bits of information relevant to setting up a mirror repo.

#### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?

#### Jetpack product discussion

Information taken from here: p1657635173682769-slack-C01U2KGS2PQ

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proof-read - and can also check the linked Slack thread to make sure the information is correct.

### CODE REVIEW

File: docs/monorepo.md
Comment (jeherve) [inline docs/monorepo.md]: Should we add extra details on the Packagist submission here? Ideally, one will want to add `automattic`, `kraft`, and `dereksmart` as maintainers after adding the package to Packagist.

Comment (coder-karen) [reply docs/monorepo.md]: I think that would be a great idea. I don't know what this involves in detail - I assume we could link to this Packagist page: https://packagist.org/about#how-to-submit-packages, then in terms of the first commit something along these lines: p1658238330799389/1658237219.448209-slack-C01U2KGS2PQ (if I understand correctly, which I don't think I am). Unless we can link to another example of what a valid `composer.json` entry should look like - perhaps one from another package. Then add those maintainers too.

Comment (jeherve) [reply docs/monorepo.md]: > in terms of the first commit something along these lines: p1658238330799389/1658237219.448209-slack-C01U2KGS2PQ (if I understand correctly, which I don't think I am). Unless we can link to another example of what a valid composer.json entry should look like - perhaps one from another package.

I personally tend to copy the new package's `composer.json` from the PR that introduces it. As I review the PR, I have all the info I need there. You do not really need to copy anything else into the mirror repo, since everything will get added once the PR is merged.

Comment (coder-karen) [reply docs/monorepo.md]: Added these details in now, thanks.

**Review by jeherve — APPROVED** [review]

**Pull request merged**
